src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+import java.time.LocalDate;
+
+public class HelloWorld {
+    private String textOut;
```

Comment:
What does `textOut` mean? Let's name your variable more informative

Reply:
I change the name of value. Pleas check "Variable textOut change" commit